src/dialog/overflow-menu/overflow-menu-pane.component.ts

```diff
@@ -32,5 +34,62 @@ export class OverflowMenuPane extends Dialog {
 		 * so we need to add some compensation)
 		 */
 		this.addGap["bottom"] = pos => position.addOffset(pos, -20, 60);
+
+		setTimeout(() => {
+			this.listItems()[0].focus();
+		});
+	}
+
+	@HostListener("keydown", ["$event"])
+	hostkeys(event: KeyboardEvent) {
+		this.escapeClose(event);
+		const listItems = this.listItems();
+
+		switch (event.key) {
+			case "ArrowDown": {
```

Comment:
Weighing in - IE11 is known as using non-standard symbols for `KeyboardEvent.key` - You may want to ensure, in case you haven't, the code works well with IE11

Reply:
IE11 will likely need a `case "Down":` in addition to this.